lifechanger/src/app/footer/footer.component.css

```diff
@@ -0,0 +1,122 @@
+a {
+  text-decoration: none;
+}
+p span:first-child {
+  background: linear-gradient(90deg, var(--blue) -2.06%, var(--pink) 100%);
+  background-clip: text;
+  background: linear-gradient(90deg, var(--blue) -2.06%, var(--pink) 100%);
+  -webkit-background-clip: text;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 26px;
+}
+p span {
+  color: var(--pink);
+}
+ul {
+  padding-left: clamp(40px, 5vw, 50px);
+  margin-bottom: 1.4rem;
+  margin-top: 1rem;
+}
+li {
+  margin-bottom: 1.3rem;
+}
+
+footer > svg {
+  margin-top: 26px;
+  margin-left: 40px;
+  width: 96%;
+}
+.nav-primary {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.nav-logos {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 130px;
+  height: 32px;
+  list-style-type: none;
+  margin: 0;
+}
+.footer-primary {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.footer-container {
+  display: flex;
+  justify-content: space-between;
+  width: 1192px;
+  height: 192px;
+  margin-top: 203px;
+  margin-bottom: -55px;
+}
+.footer2-container {
```

Comment:
Why do you have footer2-container ? What is the pupropse of it ? Is it necessary ? I see a lot of similiarity between   .footer-container and  .footer2-container. I think common styles should be moved to one class and lhere you just need to leave diferrent one. Also try to think about name of class, it's not the best way to use numbers in css class.

Reply:
Dear Mentor, I put the .footer-container into both <div> (instead of .footer-container and .footer2-container separately).
and created the .footer__nav-container (instead of rules that differs in footer2-container from .footer-container).

<div> .footer-container {   (common styles)
<div> .footer-container {   (common styles)  {  .footer__nav-container (differentiate styles that was in .footer2-container)